gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # https://kotlinlang.org/docs/releases.html#release-details
-kotlin = "1.8.21"
+kotlin = "1.8.22"
```

Comment:
Can we leave that to another PR?

Reply:
@VaiTon We can't create another PR for such a small change. Morever, this kotlin version is needed for integrating compose. Kindly check another replies too above.